README.md

```diff
@@ -696,6 +695,9 @@ series:
 
 ## Changes
 
+v4.2.3
```

Comment:
Suggest removing this section.  That we we can merge this into a new branch, and merge Steve's changes into a new branch, and both can go into a single release.  In any case, remove it will fix the clash with the existing v4.2.3.

Reply:
Yeah, probably simpler that I do that